### PR TITLE
fix: replace header and footer

### PR DESCRIPTION
**Pull request by amateima**
Signed-off-by: amateima <amatei@umaproject.org>

Bring the header and the footer in line with the new website versions.

### CODE REVIEW

File: components/footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+const LINKS_LIST = [
+  { name: "Docs", href: LINKS.docs },
+  { name: "FAQS", href: LINKS.faq },
+  { name: "Contact", href: "mailto:hello@umaproject.org" },
+  { name: "Getting Started", href: LINKS.getStarted },
+  { name: "Vote", href: LINKS.vote },
+  { name: "Careers", href: "https://angel.co/company/uma-project/jobs" },
+];
+
+const MAILCHIMP_URL =
```
Comment (daywiss) [inline components/footer/index.tsx:17]: this should probably be an env?

File: components/header/Header.tsx [hunk 2]
```diff
@@ -1,35 +1,175 @@
...
-    <Wrapper>
-      <MaxWidth>
-        <Link href="https://umaproject.org">
-          <Logo />
-        </Link>
-        <Navigation>
+    <UI.Container>
+      <UI.Content>
+        <UI.LogoLink href="/">
+          <UI.LogoLinkIcon src="/logo.svg" alt="UMA logo" />
```
Comment (daywiss) [inline components/header/Header.tsx:35]: should this go to umaproject or stay on umaverse?

**Review by daywiss — APPROVED** [review]
lgtm

File: components/header/Header.tsx (hunk 2 above)
Comment (amateima) [reply components/header/Header.tsx:35]: hmm, good question

File: components/footer/footer.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+    :nth-of-type(2) {
+      width: 45%;
+      border-left: 1px solid var(--gray-500);
+    }
+  }
+`;
+
+export const Logo = styled.img`
+  width: 100;
```
Comment (Gamaranto) [inline components/footer/footer.styled.tsx:43]: i think those require `px` or a `%`

**Comment by Tommy1231232** [comment]
@amateima looks really good. Only item I'm seeing is the FAQ page in the footer 404's. Can we update it to "FAQs" instead of "FAQS" and link to the FAQ page: https://umaproject.org/faq

**Pull request merged**